README.md

```diff
@@ -42,9 +42,9 @@ To get a local copy up and running follow these simple example steps.
 
```

Comment:
- [ ] Kindly,  It would be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. This includes `git clone URL`, `cd project_name`, etc. You can put the steps to clone and run the project in a new Getting Started section under the Built With section ✍🏻 

- [OPTIONAL] Make sure to remove all unnecessary titles since they are empty (### Run tests, ### Deployment ....) 🚮